src/app/app.component.spec.ts

```diff
@@ -1,18 +1,44 @@
-import {async, TestBed} from '@angular/core/testing';
-import {RouterTestingModule} from '@angular/router/testing';
-import {AppComponent} from 'src/app/app.component';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from 'src/app/app.component';
+import {configureTestSuite} from 'ng-bullet';
+import {PropertiesComponent} from './components/properties/properties.component';
```

Comment:
Consider using absolute import in this file please

Reply:
am working on it

Reply:
done